Fix incorrect field name (#813)
redpanda-data/documentation: modules/get-started/partials/licensing/enterprise-features.adoc

// tag::self-managed[]
* xref:manage:tiered-storage.adoc[Tiered Storage]

> * xref:manage:cluster-maintenance/continuous-data-balancing.adoc[Continuous Data Balancing]

Author: ```suggestion
* xref:manage:cluster-maintenance/continuous-data-balancing.adoc[Continuous Data Balancing]
* xref:manage:cluster-maintenance/cluster-balancing.adoc#intra-broker-partition-balancing[Intra-broker partition balancing (`core_balancing_continuous`)]
```

// tag::self-managed[]
* xref:manage:tiered-storage.adoc[Tiered Storage]
* xref:manage:cluster-maintenance/continuous-data-balancing.adoc[Continuous Data Balancing]
* xref:manage:remote-read-replicas.adoc[Remote Read Replicas]
* xref:manage:audit-logging.adoc[Audit Logging]
* xref:manage:security/fips-compliance.adoc[FIPS Compliance]
* xref:manage:schema-reg/schema-id-validation.adoc[Server-side Schema ID Validation]

> * xref:manage:security/authentication.adoc#enable-kerberos[Kerberos Authentication]

Author: ```suggestion
* xref:manage:security/authentication.adoc#kerberos[Kerberos Authentication]
```

* xref:manage:security/authentication.adoc#oidc [OAUTHBEARER/OIDC Authentication]
* xref:manage:security/authorization/rbac.adoc[Redpanda Role-Based Access Control (RBAC)]
// end::self-managed[]
// tag::console[]
* xref:console:config/security/authorization.adoc[Redpanda Console Authorization (RBAC)]
* xref:console:config/security/authentication.adoc[Redpanda Console Authentication]
// end::console[]
// tag::connect[]

> * xref:redpanda-connect:components:outputs/snowflake_put.adoc[Snowflake connector for Redpanda Connect]
> * xref:redpanda-connect:components:outputs/splunk_hec.adoc[Splunk connector for Redpanda Connect]

Author: ```suggestion
The Enterprise Edition of Redpanda Connect includes additional connectors. For a list of all available connectors, see xref:redpanda-connect:components:catalog.adoc[].
```
